packages/models-library/src/models_library/projects_nodes_io.py

```diff
@@ -191,16 +187,16 @@ def check_discriminator(cls, v):
             raise ValueError(msg)
         return 0
 
-    @validator("label", always=True, pre=True)
+    @field_validator("label", mode="before")
```

Comment:
always?

Reply:
Added `validate_default=True` to the `label` field.